Buttplug.Core/ButtplugJsonMessageParser.cs

```diff
@@ -21,38 +21,50 @@ public class ButtplugJsonMessageParser
         [NotNull]
         private readonly JsonSchema4 _schema;
 
-        public ButtplugJsonMessageParser([NotNull] IButtplugLogManager aLogManager)

```

Comment:
Making the log manager nullable is intended for the xamarin stuff , not really intended for this PR. Sorry >~<